### PR TITLE
style: quest animations

### PR DESCRIPTION
**Pull request by AjimenezDCL**
1st batch of quests UI animation.  Remember to use the `QUESTS_ENABLED` flag in the explorer.

https://app.zenhub.com/workspaces/unity-6047bad476c3c0001942ee91/issues/decentraland/unity-renderer/86

### CODE REVIEW

**Review by pbosio — APPROVED** [review]

**Review by sandrade-dcl — APPROVED** [review]
LGTM!

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsTrackerHUD/QuestsTrackerTask.cs [hunk 1]
```diff
@@ -44,9 +47,21 @@ public void Populate(QuestTask task)
             }
         }
 
-        internal void SetProgressText(float current, float end)
+        private void Update()
+        {
+            Vector3 scale = progress.transform.localScale;
```
Comment (pravusjif) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsTrackerHUD/QuestsTrackerTask.cs:52]: Would it make sense to add an escape line here ? like so:
`if (scale.x == progressTarget) return;`

**Review by pravusjif — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsTrackerHUD/QuestsTrackerTask.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsTrackerHUD/QuestsTrackerTask.cs:52]: That's a really good suggestion, gonna do it before merging thanks!

Comment (AjimenezDCL) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsTrackerHUD/QuestsTrackerTask.cs:52]: Applied

**Pull request merged**